common/src/androidTest/java/com/microsoft/identity/common/SharedPreferencesAccountCredentialCacheTest.java

```diff
@@ -1466,6 +1469,75 @@ public void getCredentialsNoHomeAccountIdNoTarget() {
         assertEquals(2, credentials.size());
     }
 
+    @Test
+    public void getCredentialsPRTNoClientId() {
+        final PrimaryRefreshTokenRecord primaryRefreshToken = new PrimaryRefreshTokenRecord();
+        primaryRefreshToken.setHomeAccountId(HOME_ACCOUNT_ID);
+        primaryRefreshToken.setEnvironment(ENVIRONMENT);
+        primaryRefreshToken.setCredentialType(CredentialType.PrimaryRefreshToken.name().toLowerCase(Locale.US));
+        primaryRefreshToken.setClientId(CLIENT_ID);
+        primaryRefreshToken.setSessionKey(SESSION_KEY);
+
+        mSharedPreferencesAccountCredentialCache.saveCredential(primaryRefreshToken);
+
+        List<Credential> credentials = mSharedPreferencesAccountCredentialCache.getCredentialsFilteredBy(
+                HOME_ACCOUNT_ID,
+                ENVIRONMENT,
+                CredentialType.PrimaryRefreshToken,
+                null, /* client id */
+                null,
+                null,
+                BEARER_AUTHENTICATION_SCHEME.getName()
```

Comment:
Auth scheme shouldn't apply here -- please confirm this test passes with `null` as its param as well. This criteria doesn't apply to PRTs

Reply:
Thanks, deleted.